e2e/atlas/helper.go

```diff
@@ -0,0 +1,117 @@
+package atlas
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+)
+
+const (
+	atlasEntity    = "atlas"
+	clustersEntity = "clusters"
+)
+
+func GetHostnameAndPort(cliPath string) (string, error) {
+	cmd := exec.Command(cliPath,
+		atlasEntity,
+		"processes",
+		"list")
+
+	cmd.Env = os.Environ()
+	resp, err := cmd.CombinedOutput()
+
+	if err != nil {
+		return "", err
+	}
+
+	var processes []*mongodbatlas.Process
+	err = json.Unmarshal(resp, &processes)
+
+	if err != nil {
+		return "", err
+	}
+
+	if len(processes) == 0 {
+		return "", fmt.Errorf("got=%#v\nwant=%#v", 0, "len(processes) > 0")
+	}
+
+	// The first element may not be the created cluster but that is fine since
+	// we just need one cluster up and running
+	return processes[0].Hostname + ":" + strconv.Itoa(processes[0].Port), nil
+}
+
+// ExistCluster returns true if there is at least a cluster is deployed, false otherwise
+func ExistCluster(cliPath string) bool {
```

Comment:
[nit] since you don't search for a particular cluster
```suggestion
func AnyCluster(cliPath string) bool {
```